xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -132,6 +156,35 @@ public ConfigurationParser(String xml) throws IOException, SAXException, JAXBExc
     DocumentBuilder domBuilder = factory.newDocumentBuilder();
     domBuilder.setErrorHandler(new FatalErrorHandler());
     Element dom = domBuilder.parse(xml).getDocumentElement();
+
```

Comment:
Maybe we can get away without it for now: https://jira.spring.io/browse/SPR-4953